### DISCUSSION TITLE
app is building but nit running

### DISCUSSION THREAD

**Discussion by guyulmaz** [skiptools/skip#209] [General]
Hi , I'm new to ios dev. I created new app but , when I click run button at xcode, it is only building but not starting simulator.
What can be reason ?

**Comment by marcprux** [comment]
Are there any errors or warnings in the Xcode report navigator tab? And is the Android emulator currently running, as per https://skip.tools/docs/gettingstarted/#creating-a-dual-platform-app ?

**Reply by guyulmaz** [reply]
It is just saying built successful
yes both ios and android simulator is open.
![Image](https://github.com/user-attachments/assets/df5b338b-b6ae-46e3-99ba-1b39e32ebcfb)

**Comment by marcprux** [comment]
It looks like you might just be building the framework target. If you select the app target from the Product/Scheme menu, can you try running from there?

**Reply by guyulmaz** [reply]
I added MyApp01App scheme, but again when pressed run only built happening.

![image](https://github.com/user-attachments/assets/7b7f18d0-7db2-4b8f-9923-86efcecafcb5)

**Comment by marcprux** [comment]
Can you show me the list of schemes (from the Manage Schemes… button)?

And when you run it, is it running on the iPhone Simulator?

**Comment by guyulmaz** [comment]
![image](https://github.com/user-attachments/assets/02472c6b-286b-4136-a0be-b26beef9a414)
![image](https://github.com/user-attachments/assets/78ba5802-e39a-4ad7-9462-dcf8eec8d6a3)

no only say built successful, then nothing happens to iphone simulator

**Comment by guyulmaz** [comment]
guyulmaz@Goksels-MacBook-Air MyApp01 % skip checkup
[✓] Skip version 1.0.7 (= 1.0.7)
[✓] macOS version 14.6.1 (> 13.5.0)
[✓] macOS architecture: ARM
[✓] Swift version 5.10 (> 5.9.0)
[✓] Xcode version 15.4 (> 15.0.0)
[✓] Xcode tools SDKs: 5
[✓] Homebrew version 4.3.17 (> 4.1.0)
[✓] Gradle version 8.10 (> 8.6.0)
[✓] Java version 22.0.2 (> 21.0.0)
[✓] Android Debug Bridge version 1.0.41 (> 1.0.40)
[✓] Android Studio version: 2024.1
[✓] Android tools SDKs: 3
[✓] Resolve dependencies (334.11s)
[✓] Build hello-skip (13.1s)
[✓] Test Swift (6.84s)
[✓] Test Kotlin (267.84s)
[✓] Archive iOS ipa (31.07s)
[✓] Assemble HelloSkip-release.ipa 38 KB
[✓] Verify HelloSkip-release.ipa 38 KB
[✓] Assembling Android apk (228.14s)
[✓] Verify HelloSkip-release.apk 13.2 MB
[✓] Check Swift Package (0.6s)
[✓] Check Skip Updates: 1.0.7
[✓] Skip 1.0.7 checkup succeeded in 894.66s

**Comment by marcprux** [comment]
I'm not sure what went wrong, but can you try creating a new project with:

```
skip init --open-xcode --appid=hello.skip hello-skip HelloSkip
```

The resulting app should have a "HelloSkip" target, which, when you run it, should show up on both the iOS Simulator and Android Emulator. Let us know if it doesn't work for you, and we can try to debug.

<img width="1512" alt="Screenshot 2024-08-25 at 14 48 18" src="https://github.com/user-attachments/assets/15f2ddc4-4683-48ba-9fa0-a9a069b1450a">

**Comment by guyulmaz** [comment]
@marcprux I had created app with
skip init --free --appid=some.free.App project-name AppName
just for your info

**Comment by guyulmaz** [comment]
![image](https://github.com/user-attachments/assets/ec387acb-4145-4b29-b3c5-9cef60adf613)

with new project it worked :)

what is problem with --free one ?

should I convert it it free one like =

add LICENSE.GPL file to root manually and 
The following source code header comment will be sufficient attestation for a .swift or .kt file to be used in free mode with the LGPL:

// This is free software: you can redistribute and/or modify it
// under the terms of the GNU Lesser General Public License 3.0
// as published by the Free Software Foundation https://fsf.org

**Comment by marcprux** [comment]
I'm not sure what the issue was, but I don't think it has anything to do with the `--free` argument. You should also be able to make a new project with:

```
skip init --free --open-xcode --appid=hello.skip hello-skip HelloSkip
```

FTR, all `--free` does it add the license file and headers necessary for Skip to not require a valid license.

**Comment by guyulmaz** [comment]
I created a new one with
skip init --free --open-xcode --appid=hello.skip hello-skip-free HelloSkip
and this worked also.
I dont know why the first one happened like that.

Thanks for the help
